spec/src/main/asciidoc/messaging-spec.adoc

```diff
@@ -102,84 +99,84 @@ _topic_. The client that sends the message _publishes_ it to a specific
 topic. The message is then delivered to all the clients that are
 _subscribed_ to that topic.
 
-==== What JMS does not include
+==== What Jakarta Messaging does not include
 
-JMS does not address the following functionality:
+Jakarta Messaging does not address the following functionality:
 
 * Load balancing/fault tolerance - Many products provide support for
-multiple, cooperating clients implementing a critical service. The JMS
+multiple, cooperating clients implementing a critical service. The Jakarta Messaging
 API does not specify how such clients cooperate to appear to be a
 single, unified service.
 * Error/advisory notification - Most messaging products define system
 messages that provide asynchronous notification of problems or system
-events to clients. JMS does not attempt to standardize these messages.
-By following the guidelines defined by JMS, clients can avoid using
+events to clients. Jakarta Messaging does not attempt to standardize these messages.
+By following the guidelines defined by Jakarta Messaging, clients can avoid using
 these messages and thus prevent the portability problems their use
 introduces.
-* Administration - JMS does not define an API for administering
+* Administration - Jakarta Messaging does not define an API for administering
 messaging products.
-* Security - JMS does not specify an API for controlling the privacy and
+* Security - Jakarta Messaging does not specify an API for controlling the privacy and
 integrity of messages. It also does not specify how digital signatures
-or keys are distributed to clients. Security is considered to be a JMS
+or keys are distributed to clients. Security is considered to be a Jakarta Messaging
 provider-specific feature that is configured by an administrator rather
-than controlled via the JMS API by clients.
-* Wire protocol - JMS does not define a wire protocol for messaging.
-* Message type repository - JMS does not define a repository for storing
+than controlled via the Jakarta Messaging API by clients.
+* Wire protocol - Jakarta Messaging does not define a wire protocol for messaging.
+* Message type repository - Jakarta Messaging does not define a repository for storing
 message type definitions and it does not define a language for creating
 message type definitions.
 
-==== Java SE and Java EE support
+==== Java SE and Jakarta EE support
 
-The JMS API is designed to be suitable for use by both Java client
+The Jakarta Messaging API is designed to be suitable for use by both Java client
 applications using the Java™ Platform, Standard Edition (Java SE), and
 Java middle-tier services using the Java™ Platform, Enterprise Edition
```

Comment:
2 occurrences of Java™ Platform, Enterprise Edition needs to be changed to Jakarta EE Platform.

Reply:
I've fixed those references. 

I also removed references to specific versions of Jakarta EE platform or Enterprise Beans as they obsolete with every new version. They were confusing anyway because they referred to an older version of Java EE because the JMS spec was finalized before the version of Java EE spec that included it. If we don't use the exact version, it should be implicitly the Jakarta EE version that includes this version of Jakarta Messaging.